### PR TITLE
Add calculator, no funcionality.

### PR DESCRIPTION
**Pull request by teqnickels**

### CODE REVIEW

File: views/index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      </div>
+      <div class="calculator-button" id="percent">
+        <button>%</button>
+      </div>
+      <div class="calculator-button" id="divide">
+        <button class="orange-button white-operator">/</button>
+      </div>
+    </div>
+
+    <div class="calculato-row2">
```
Comment (jrobcodes) [inline views/index.html:30]: Typo - `calculato` instead of `calculator`

File: views/index.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  <meta charset ="utf-8">
+  <title>Benchmark Calculator</title>
+</head>
+<body>
+
+  <div class="calculator-container">
+    <div id="display">
+    </div>
+
+    <div class="calculator-row1">
```
Comment (jrobcodes) [inline views/index.html:15]: Consider renaming this to simply `calculator-row` - if you need to apply styles based on the order of rows (1st, 2nd, etc.), you may target these with the `:nth-child(n)` selector

File: views/index.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+<html>
+<head>
+  <link rel="stylesheet" type="text/css" href="stylesheet.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <meta charset ="utf-8">
+  <title>Benchmark Calculator</title>
+</head>
+<body>
+
+  <div class="calculator-container">
```
Comment (jrobcodes) [inline views/index.html:11]: Consider renaming this to describe the content, not the markup - `calculator` instead of `calculator-container`.

File: views/stylesheet.css [hunk 4]
```diff
@@ -0,0 +1,59 @@
+.calculator-container {
+  width: 232px;
+  height: 323px;
+  border-radius: 15px;
+  position: absolute;
```
Comment (jrobcodes) [inline views/stylesheet.css:5]: The specs require this to be centered in the page - the absolute positioning with top, bottom, left, and right set to 0 will stretch this to fill the page. (Also, given this positioning, the `margin: auto` will be ignored.)

File: views/stylesheet.css [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  top:0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  margin: auto;
+}
+
+#display {
```
Comment (jrobcodes) [inline views/stylesheet.css:13]: Avoid the use of IDs, unless needed for form labels `<label for="element-id">`.  This should also be namespaced under `calculator`, as defined in the specs: `.calculator-display`

File: views/stylesheet.css [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  height: 75px;
+  background-color: #8C8C8C;
+}
+
+.calculator-button {
+  display:inline;
+  width: 232px;
+}
+
+.orange-button {
```
Comment (jrobcodes) [inline views/stylesheet.css:24]: These styles should be namespaced as well: `.calculator-button-orange`.  Note that I inverted the order as well, to remain consistent with the other elements in this namespace (i.e. to show that this color applied to `.calculator-button`s).

**Review by jrobcodes — COMMENTED** [review]
There are a few naming conventions from the spec that are not honored - I've left notes inline to call these out.

File: views/stylesheet.css (hunk 4 above)
Comment (teqnickels) [reply views/stylesheet.css:5]: I believe because I have the size of the calculator constricted to specific and explicit dimensions that this prevents the calculator from stretching to the size of the screen. Also, when I remove the margin : auto, the calculator is no longer centered.  Actually removing any of the positioning styles removes the calculator from the center of the page

Comment (jaredatron) [reply views/stylesheet.css:5]: @teqnickels this is an interesting solution to the center/center problem. This works in some ways but fails in some others. This does in fact center/center the calculator but `position: absolute` causes the calculator to float above the page. This causes the calculator to not take up any space. So you'll notice that the body element on your page has 0 height. 

TL;DR: Using `position: absolute` in combination with `margin: auto` is an interesting trick but I strongly encourage you to look into flexbox.

Comment (teqnickels) [reply views/stylesheet.css:5]: Thanks for explaining what's happening with this style. Since I can't "see" that effect when I'm looking at the calculator in my browser, it was hard to see that it wasn't the best option. Also, I totally thought flexbox was a library, LOL.
